client-participation/vis2/components/exploreTid.js

```diff
@@ -69,21 +69,21 @@ const DataSentence = ({math, selectedTidCuration, selectedComment, repfulFor, St
     s = s.replace("{{comment_id}}", selectedComment.tid);
 
     markup = (
-      <div style={{display: "flex"}}>
+      <div style={{ display: "flex" }}>
         <svg height={40} style={{
-            display: "inline",
-            marginRight: 10,
-            fill: repfulForAgree ? globals.colors.agree : globals.colors.disagree
-          }} viewBox="0 0 1792 1792">
-          <path d={repfulForAgree ? checkmark : ban}/>
-          </svg>
+          display: "inline",
+          marginRight: 10,
+          fill: repfulForAgree ? globals.colors.agree : globals.colors.disagree
+        }} viewBox="0 0 1792 1792">
+          <path d={repfulForAgree ? checkmark : ban} />
+        </svg>
         <p style={{
-            fontSize: 14,
-            fontFamily: "Helvetica",
-            fontWeight: 500,
-            maxWidth: 240,
-            color: "rgb(180,180,180)"
-          }}>
+          fontSize: 14,
+          fontFamily: "Helvetica",
+          fontWeight: 500,
+          maxWidth: 240,
+          color: "rgb(117,117,117)"
+        }}>
```

Comment:
nit: pls revert (will explain later)
```suggestion
            fontSize: 14,
            fontFamily: "Helvetica",
            fontWeight: 500,
            maxWidth: 240,
            color: "rgb(117,117,117)"
          }}>
```